package.json

```diff
@@ -54,9 +54,9 @@
     "jest": "^24.9.0",
     "jest-cli": "^24.9.0",
     "kremling": "^2.0.1",
+    "lodash": "^4.17.15",
```

Comment:
Why the switch from lodash-es to lodash?

lodash-es allows for [tree shaking](), whereas lodash does not. Can you compare the bundle sizes when you use `lodash` vs `lodash-es` (run `npm run build` and check file sizes in `dist` directory).

My guess is that the switch to lodash increased the bundle size substantially. https://webpack.js.org/guides/tree-shaking/#root

Reply:
With lodash:

<img width="597" alt="image" src="https://user-images.githubusercontent.com/5524384/66763446-0edd4300-ee65-11e9-95ea-098b8e4097a9.png">

With lodash-es:
<img width="595" alt="image" src="https://user-images.githubusercontent.com/5524384/66763477-1c92c880-ee65-11e9-8724-3bc6228c9c9b.png">

Reply:
restored lodash-es and updated the branch